chainerrl/links/noisy_linear.py

```diff
@@ -26,10 +28,22 @@ def __init__(self, mu_link, sigma_scale=0.4):
 
         with self.init_scope():
             self.mu = mu_link
+            self.mu.W.initializer = VarianceScalingUniform(1)
+            if not self.nobias:
+                self.mu.b.initializer = VarianceScalingUniform(1)
+
+            # assure weight reinitialization
```

Comment:
I feel reinitialization is tricky.  How about creating a new `L.Linear` for `self.mu` (like the interface before the commit https://github.com/chainer/chainerrl/pull/169/commits/6af88d63c97b0b6e36eedc5c7e03190ec314cb52#diff-3d0a1daaad7846c7235e591fbf316217)?